src/data/json_to_parq.py

```diff
@@ -48,15 +48,15 @@ def clean_for_parquet(df):
 
 def parquet_to_gcs(df, day):
     df.to_parquet(
-        f'gs://thepanacealab_covid19twitter/dailies/'
+        'gs://thepanacealab_covid19twitter/dailies/'
         + f'{day}/{day}_tweets.parquet'
```

Comment:
This is weird formatting. Consider putting this on one line and making the whole thing a format string. If flake8 yells at you, you should change the line character limit to 100 or higher, as I mentioned (default is like 70).

Reply:
Added `.flake8` config file to repo root, that increases the line limit to 100 (and ignores files within `experiment` folder).